lib/openstack/openstack_event_monitor.rb

```diff
@@ -10,10 +10,13 @@ def self.new(options={})
   end
 
   def self.available?(options)
-    !select_event_monitor_class(options).kind_of? OpenstackNullEventMonitor
+    select_event_monitor_class(options) != OpenstackNullEventMonitor
   end
 
-  DEFAULT_PLUGIN_PRIORITY = 0
+  PLUGIN_PRIORITY_HIGH    = 10
+  PLUGIN_PRIORITY_MED     = 5
+  PLUGIN_PRIORITY_LOW     = 1
+  DEFAULT_PLUGIN_PRIORITY = PLUGIN_PRIORITY_LOW
```

Comment:
Should we make it so a lower number is higher priority?  This is consistent with other priority things in our code base (at least the ones I know of).